docs/user/ppl/functions/collection.rst

```diff
@@ -394,3 +394,53 @@ Example::
     | [alex,celestino,claudia] |
     +--------------------------+
 
+
+MVZIP
+-----
+
+Description
+>>>>>>>>>>>
+
+Usage: mvzip(mv_left, mv_right, [delim]) combines the values in two multivalue fields. The delimiter is used to specify a delimiting character to join the two values. This is similar to the Python zip command.
+
+The values are stitched together combining the first value of mv_left with the first value of mv_right, then the second with the second, and so on. The function stops at the length of the shorter field.
+
+The delimiter is optional. When specified, it must be enclosed in quotation marks. The default delimiter is a comma ( , ).
+
+Argument type: mv_left: ANY, mv_right: ANY, delim: STRING (optional)
+
+Return type: ARRAY
+
+Example::
+
+    os> source=people | eval hosts = array('host1', 'host2'), ports = array(80, 443), nserver = mvzip(hosts, ports) | fields nserver | head 1
```

Comment:
np: `head 1` is because `source=people` is required? Do we have an issue for adding `makeresults` command which would simplify this a lot?